tests/load/sender.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/core/types"
+	"github.com/ava-labs/libevm/ethclient"
+
+	"github.com/ava-labs/avalanchego/ids"
+
+	ethereum "github.com/ava-labs/libevm"
+)
+
+type Builder interface {
+	// Create a valid transaction
+	BuildTx() (*types.Transaction, error)
+	// Increment the nonce of the issuer by 1
+	IncrementNonce()
+}
+
+type Sender struct {
+	client *ethclient.Client
+}
+
+func NewSender(client *ethclient.Client) *Sender {
+	return &Sender{client: client}
+}
+
+func (w *Sender) SendTx(
+	ctx context.Context,
+	builder Builder,
```

Comment:
This function signature implies to me that we could have one sender with multiple builders... would it make more sense to just accept a `Builder` in the constructor instead of having it as a function param here?